src/components/MobileApp/Wireframes/Components/OrderList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import Section from './Section'
+import OrderStateInfo from './OrderStateInfo'
+import ListItem from './ListItem'
+
+export default function OrderList({ orders, onItemClick }) {
+  return (
+    <div>
+      {Object.entries(orders).map(([group, orders]) => {
```

Comment:
Hier besser explizit über alle möglichen "group" states mappen. Kann aber erstmal so bleiben, vielleicht sollten die Daten in anderer Struktur kommen.
* Problem mit dieser Lösung: Die Reihenfolge der keys ist (theoretisch) nicht stabil
* Gut an der Lösung: UI muss nicht alle Zustände kennen